web-ui/src/main/resources/catalog/components/usersearches/UserSearchesDirective.js

```diff
@@ -89,6 +89,12 @@
           '../../catalog/components/usersearches/partials/usersearchespanel.html',
         link: function postLink(scope, element, attrs) {
           scope.lang = gnLangs.current;
+          // Configure the base url to launch the user search selected (search or board page)
+          if ($route && $route.current && $route.current.$$route.originalPath === '/board') {
```

Comment:
Testing in more details, this does not work ? TypeError: Cannot read property 'originalPath' of undefined
Any idea ?

Reply:
It happens when choosing a user search